### DISCUSSION TITLE
Aspect slider's min and max values

### DISCUSSION THREAD

**Discussion by Khavrolev** [nanxiaobei/antd-img-crop#278] [General]
You provide API for `minZoom` and `maxZoom`, but why you don't provide the similar props for aspect's slider? Currently I set aspect by image's proportions by default using `beforeCrop`, and this default aspect could be more than 2 or less than 0.5. That's why I have problems with the aspect's slider

![image](https://github.com/nanxiaobei/antd-img-crop/assets/43345369/4e0ad915-267f-4076-a59d-5de9b80cb2f7)
![image](https://github.com/nanxiaobei/antd-img-crop/assets/43345369/fa7933bb-9f12-4be7-8e93-00a17bac826c)
![image](https://github.com/nanxiaobei/antd-img-crop/assets/43345369/4adb7101-5e25-492c-b7b5-59520c385c43)

**Comment by nikhilnaikdelvify** [comment]
I have the same issue, any work around on increasing aspect ratio in slider, always between 0.5 and 2.0, even though sometimes ou images can be like 9.73858 or something

**Comment by nikhilnaikdelvify** [comment]
@nanxiaobei any response???

**Comment by coding-parsa** [comment]
I have this issue too did anyone had a solution ?

**Comment by nanxiaobei** [comment]
added `minAspect` & `maxAspect` in `4.24.0`.
